mysql-test/suite/rpl/t/rpl_bug85158.test

```diff
@@ -0,0 +1,108 @@
+#
+# Testcase for bug 85158.
+#
+--source include/not_group_replication_plugin.inc
+--source include/have_gtid.inc
+--source include/have_binlog_format_row.inc
```

Comment:
Does the bug reproduce under row only or is it simply binlog format-independent? In the latter case let's just run it under all three formats unless its runtime is very long

Reply:
yep

Reply:
"yep" as in "row only"? The testcase is still limited to that format